docs/custom-flows/error-handling.mdx

```diff
@@ -1,5 +1,100 @@
 ---
-sanity_slug: /docs/authentication/error-handling
+title: Error handling
+description: Provide your users with useful information about the errors being returned from Sign In and Sign Up requests.
```

Comment:
I assume this is an intentional change; but just want to call it out to firm up my understanding.  It looks like we used to render this description on the page.  Assume this now is only used for the <title> meta tags?

![CleanShot 2023-09-21 at 12 24 40@2x](https://github.com/clerkinc/clerk-docs/assets/2105067/1e2ddad8-0b0c-4660-89b9-5f8466190de5)

Reply:
Sanity descriptions are not used in MDX. They are used instead in our SEO descriptions.